beta/nncf/tensorflow/sparsity/magnitude/algorithm.py

```diff
@@ -172,18 +158,15 @@ def _collect_all_weights(self):
         all_weights = []
         for wrapped_layer in collect_wrapped_layers(self._model):
             for weight_attr, ops in wrapped_layer.weights_attr_ops.items():
-                for op in ops.values():
-                    if isinstance(op, BinaryMask):
+                for op_name in ops:
+                    if op_name in self.op_names:
                         all_weights.append(tf.reshape(
                             self.weight_importance(wrapped_layer.layer_weights[weight_attr]),
                             [-1]))
         return all_weights
 
-    def statistics(self, quickly_collected_only=False):
-        raw_sparsity_statistics = self.raw_statistics()
-        prefix = 'sparsity'
-        header = ['Name', 'Weight\'s Shape', 'SR', '% weights']
-        return convert_raw_to_printable(raw_sparsity_statistics, prefix, header)
+    def statistics(self, quickly_collected_only: bool = False):
+        return super().statistics(quickly_collected_only=quickly_collected_only)
```

Comment:
Seem like this method is not needed here because it uses implementation from the superclass only.

Reply:
Removed it